docs/articles/getting-started/execution.md

```diff
@@ -109,6 +109,7 @@ def input_builder(worker_index, worker_count, resume_epoch):
         yield AdvanceTo(epoch)
         yield Emit({"input_from": worker_index, "x": x})
 
+input_config = ManualInputConfig(input_builder)
```

Comment:
Would this be cuter as a python input helper? So we could inline `manual_config(input_builder)` into the `spawn_cluster` call? Or do we want to keep config choices overt for now? (that's where i'm leaning atm tho it's kinda clunky)